src/executor/analyze_executor.cpp

```diff
@@ -0,0 +1,69 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// analyze_executor.cpp
+//
+// Identification: src/executor/analyze_executor.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <vector>
+
+#include "executor/analyze_executor.h"
+#include "executor/executor_context.h"
+#include "concurrency/transaction_manager_factory.h"
+#include "common/logger.h"
+#include "catalog/catalog.h"
+#include "optimizer/stats/stats_storage.h"
+
+namespace peloton {
+namespace executor {
+
+AnalyzeExecutor::AnalyzeExecutor(const planner::AbstractPlan *node,
+                                 ExecutorContext *executor_context)
+    : AbstractExecutor(node, executor_context),
+      executor_context_(executor_context) {}
+
+bool AnalyzeExecutor::DInit() {
+  LOG_TRACE("Initializing analyze executor...");
+  LOG_TRACE("Analyze executor initialized!");
+  return true;
+}
+
+bool AnalyzeExecutor::DExecute() {
+  LOG_TRACE("Executing Analyze...");
```

Comment:
@rohit-cmu Right, this is added just for consistency with other executor implementations (`create_executor`, `drop_executor` etc). A LOG_TRACE won't hurt in this case.